docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       - 5000
     volumes:
       - .:/code
+      - images:/images
```

Comment:
Why do you need to create another volume?

Reply:
I didn't realize `.:/code` gave me access to what I needed. I got it working now